preview_renderer.js

```diff
@@ -4,91 +4,97 @@ const $ = require('jquery');
 const nsh = require('node-syntaxhighlighter');
 const htmlFormatter = require('js-beautify').html;
 
-
-const assetMap = main.assetMap
-const hatchFolder = main.hatchFolder
+const {assetMap, hatchFolder} = main;
 
 let showingSource = false;
 
-$(document).ready(function(){
-  $('#flip_controls').hide();
-  $('#source_code').hide();
-  $('#image_asset').hide();
-})
-
-flipPage = function() {
-  showingSource = !showingSource;
-  if (showingSource) {
-    $('#preview_frame').hide();
-    $('#image_preview').hide();
-    $('#source_code').show();
-  } else {
-    $('#preview_frame').show();
-    $('#image_preview').show();
+$(document).ready(() => {
+    $('#flip_controls').hide();
     $('#source_code').hide();
-  }
-}
+    $('#image_asset').hide();
+});
 
-setPreviewAssetID = function(ID) {
-  if (!ID) {
-    $('#preview_frame').html("<center><h2>No asset selected</h2></center>")
-    return;
-  }
+window.flipPage = function () {
```

Comment:
Do we need all of these internal methods attached to the window? I'm seeing this in other files too.

Reply:
At least `preview` and `flipPage` have to be defined on `window`, since they're used from `onclick` handlers in the HTML.

Reply:
I fixed the other ones to use `exports` and have the modules where they're used `require` them directly.

Reply:
:+1: